modular_skyrat/code/modules/cargo/packs/medical.dm

```diff
@@ -0,0 +1,36 @@
+/datum/supply_pack/medical/firstaidmixed
+	name = "Mixed Medical Kits"
+	desc = "Contains one of each medical kits for dealing with a variety of injured crewmembers."
+	cost = 2000
+	contains = list(/obj/item/storage/firstaid/toxin,
+					/obj/item/storage/firstaid/o2,
+					/obj/item/storage/firstaid/brute,
+					/obj/item/storage/firstaid/fire,
+					/obj/item/storage/firstaid/regular)
+	crate_name = "medical kit crate"
+
```

Comment:
There's one missing, I got the item path for you below:
/obj/item/storage/firstaid/radbgone
Ya' can do this to quickly edit stuff in if you haven't found out already:
![image](https://user-images.githubusercontent.com/53862927/88421653-d483d780-cde0-11ea-8f3a-94e86f96878a.png)

Reply:
This is actually a restoration of a package that was deleted upstream (along with all other medical kit packages). This is a standard set, there is already a radiation bundle covered in the non-modular medicine tab.